cmake/std/PullRequestLinuxDriverMerge.py

```diff
@@ -156,20 +146,15 @@ def run():
             return_value = False
         except subprocess.CalledProcessError as cpe:
             return_value = False
-            print('Recieved subprocess.CalledProcessError - returned {error_num}'.format(error_num=cpe.returncode),
-                  file=sys.stdout)
-            print('  from command {cmd}'.format(cmd=cpe.cmd),
-                  file=sys.stdout)
-            print('  output {out}'.format(out=cpe.output),
-                  file=sys.stdout)
+            print('Recieved subprocess.CalledProcessError - returned {error_num}'.format(error_num=cpe.returncode))
```

Comment:
Same here - stderr

Reply:
@prwolfe is it just these two that you mention here that should have been STDERR?

Do you know if changing them to STDERR might cause them to _not_ get captured to the autotester when it pulls the console log (i.e., does it only capture STDOUT or does it get both STDOUT + STDERR?)

Reply:
There were two of these. Im insure if this would affect the attester logs or not. As such leave it as is - I was just commenting that "generally" error messages should not go to stdout.

Glad you thought about that downstream question.

Reply:
Sure.  It's something that I ran into with SPiFI and log processing. Internally Jenkins pipelines only capture the output to STDOUT when you launch a command from their `sh()` instruction in a pipeline... This is nice and all, but people look at the console output when something goes wrong and to _not_ capture STDOUT+STDERR limits the utility of that.  Splitting STDERR from STDOUT can also be less than useful when looking at logs because often it's really helpful to see WHERE the error message is within the context of the overall execution :p 

Anyways, I re-added the `file=sys.stodut` entries for those print commands in the Merge file so they're there and we can change them to stderr down the line of we want to after doing a bit of checking on what the AT does, etc. :)